src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -3447,6 +3447,86 @@ public static void setConsecutiveMessageErrorsThreshold(int value)
         conf.consecutive_message_errors_threshold = value;
     }
 
+    public static boolean enablePartitionDenylist()
+    {
+        return conf.enable_partition_denylist;
```

Comment:
We need to update `cassandra.yaml` with mentioning all the properties and their description.